flatpak/build.sh

```diff
@@ -1,8 +1,22 @@
 #!/usr/bin/env bash
 
-# SPDX-License-Identifier: GPL-3.0+
```

Comment:
This file has an SPDX-License header and already a copyright notice, no need to add the full license too

Reply:
Changed to only update year range of existing copyright line, removed the full license text.